.pre-commit-config.yaml

```diff
@@ -12,8 +12,8 @@ repos:
     - --baseline
     - .secrets.baseline
     id: detect-secrets
-  repo: https://github.com/Yelp/detect-secrets
-  rev: v1.5.0
+  repo: https://github.com/ibm/detect-secrets
```

Comment:
The reason I didn't use the IBM one for a few reasons:

1. The code here hasn't been updated in two years.
2. An issue where it seems that Python 3.12 is not supported. 3.12 is the Python used by SRE Scenarios.